client/actions/events/api.js

```diff
@@ -653,6 +653,10 @@ const query = (
     (dispatch, getState, {api}) => {
         let must = [];
 
+        if (selectors.multiSelect.isTotalSelected(getState())) {
+            maxResults = selectors.main.eventsTotalItems(getState()); // eslint-disable-line no-param-reassign
```

Comment:
`API` is configured to only return max 200 items. Hence you won't get more than 200 items. Moreover, `refetch` action only fetches based on the page number set in the store.